spec/features/users/sign_in_spec.rb

```diff
@@ -1,49 +1,30 @@
-# Feature: Sign in
-#   As a user
-#   I want to sign in
-#   So I can visit protected areas of the site
-feature 'Sign in', :devise do
-
-  # Scenario: User cannot sign in if not registered
-  #   Given I do not exist as a user
-  #   When I sign in with valid credentials
-  #   Then I see an invalid credentials message
+feature 'Sign in', %{
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.